src/extension/devtools/devtools.ts

```diff
@@ -131,6 +134,13 @@ async function createDevtoolsPanel() {
   panel.onShown.addListener((window) => {
     panelWindow = getPanelActor(window);
 
+    unsubscribers.add(
+      createRPCBridge(
+        createPortMessageAdapter(port),
+        createWindowMessageAdapter(window)
+      )
+    );
+
```

Comment:
Not 100% sure if this is the right place to create this bridge 😬

Reply:
I would make two changes here:

1. Move the `createRPCBridge` call inside the the `if (!connectedToPanel)` below.
2. Remove the `unsubscribers.add` call around this

The reasoning here is that we still want to allow messages to travel between the background port and the panel window even when the panel is hidden (e.g. the user navigated away by clicked the "Elements" tab). I can think of some cases like opening the Apollo tab, then quickly switching away from it, but still allowing the injected script to report back whether it found a client or not (which will eventually be an RPC call as well). If we unsubscribe the bridge, any message that gets sent while the panel is hidden will be dropped.

The `if (!connectedToPanel)` should only ever execute the first time the panel is created, so we shouldn't have to worry about this script accidentally creating too many bridges.